tendrl/commons/flows/__init__.py

```diff
@@ -4,8 +4,8 @@
 from tendrl.commons.event import Event
```

Comment:
Do you need `Event`?

Reply:
yes, because ExceptionMessage class is using Event, for that we can't use log_util

Reply:
in this file ExceptionMessage is used so i am keeping Event

Reply:
ok